installing/installing-fips.adoc

```diff
@@ -0,0 +1,74 @@
+
+Because FIPS must be enabled before the operating system that your cluster uses boots for the first time, you cannot enable FIPS after you deploy a cluster.
+
+[id="installation-about-fips-validation_{context}"]
+== FIPS validation in {product-title}
+
+{product-title} uses certain FIPS validated modules within Red Hat Enterprise Linux (RHEL) and {op-system} for the operating system components that it uses. See link:https://access.redhat.com/articles/3359851[RHEL7 core crypto components]. For example, when users SSH into {product-title} clusters and containers, those connections are properly encrypted.
+
+{product-title} components are written in Go and built with Red Hat’s golang compiler. When you enable FIPS mode for your cluster, Red Hat’s golang compiler calls RHEL and {op-system} cryptographic libraries for all {product-title} components that require cryptographic signing. At the initial release of {product-title} version 4.3, only the `ose-sdn` package uses the native golang cryptography, which is not FIPS validated. Red Hat verifies that all other packages use the FIPS validated OpenSSL module.
+
+.FIPS mode attributes and limitations in {product-title} {product-version}
+[cols="8a,8a",options="header"]
+|===
+
+|Attributes
+|Limitations
+
+|FIPS compliant operating systems: RHEL 7 and {op-system} 8.
+.3+|The FIPS implementations lacks use of single function computing hash and validating keys based on compliant run times and services. This limitation will continue to be evaluated and improved in future {product-title} releases.
+
+|FIPS compliant CRI-O runtimes.
+|FIPS compliant {product-title} services.
+
+|FIPS validated cryptographic module and algorithms that are obtained from RHEL 7 and {op-system} binaries and images.
+|
+
+|Use of FIPS compatible golang compiler.
+|TLS FIPS compliance is not complete but is planned for future {product-title} releases.
+
+|===
+
+[id="installation-about-fips-components_{context}"]
+==  FIPS compliance in components that the cluster uses
+
+Although the {product-title} cluster itself uses FIPS validated modules, ensure that the systems that support your {product-name} cluster use FIPS validated modules for cryptography.
+
+[id="installation-about-fips-components-etcd_{context}"]
+=== etcd
+
+To ensure that the secrets that are stored in etcd use FIPS validated encryption, encrypt the etcd volume by using RHEL-provided disk encryption.
```

Comment:
What do you really want to talk about? The disk encryption or encryption of secrets stored in etcd(in story https://jira.coreos.com/browse/MSTR-485)?If you are talking about disk encryption, will you add the encryption of secrets stored in etcd? Thanks. Besides, OCP QE didn't do special test to cover disk encryption.

Reply:
@knewcomerRH, which angle do you want to cover?

Reply:
@kalexand-rh I would not mention the ability to encrypt the etcd datastore or to do NBDE in the FIPS section of the documentation. Just focus on FIPS. @simo5 what can we say about FIPS and the aes cbc encryption algorithm used to encrypt the etcd datastore?

Reply:
AES CBC is an approved algorithm so it should be fine ?
Not sure I understand the context thugh.

Reply:
For now, I'm adding a link to the process for encrypting the etcd datastore: http://file.rdu.redhat.com/kalexand/120519/osdocs617/authentication/encrypting-etcd.html#encrypting-etcd

Reply:
@kalexand-rh We should focus on encrypting the etcd datastore, not the RHCOS volumes here. And the aes cbc crypto algorithm which we use is FIPS approved.

Reply:
Thank you @knewcomerRH! I've updated this section to discuss the etcd datastore instead. Please let me know if you see anything else that needs to change.